### PR TITLE
adding gitter room to documentation issue #208

### PR DESCRIPTION
**Pull request by jmbradnan**
Added link for C# Gitter room to Social Media section of docs/RESOURCES.md file

### CODE REVIEW

File: docs/RESOURCES.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 * [StackOverflow ](http://stackoverflow.com/questions/tagged/c%23) can be used to search for your problem and see if it has been answered already. You can also ask and answer questions.
 * [/r/csharp](https://www.reddit.com/r/csharp) is the C# subreddit.
 * [@dotnet](https://twitter.com/DotNet) is the official .NET Twitter account.
+* [Gitter](https://gitter.im/exercism/xcsharp) is the C# Gitter room.
```
Comment (ErikSchierboom) [inline docs/RESOURCES.md]: Perhaps we can reword this slightly, to make it more clear that people can ask questions about exercism's C# track here. Maybe something like: 

"For questions about the Exercism C# track, please use the [Gitter room](https://gitter.im/exercism/xcsharp)."

Or:

"To get support for the C# track, please use the [Gitter room](https://gitter.im/exercism/xcsharp)."

Or perhaps something else entirely. What do you feel is the best option?

The main point is that we should explain why people should go to the Gitter room. Sorry I didn't make that clear in the issue.

Comment (ErikSchierboom) [inline docs/RESOURCES.md]: I really like the description used in the F# version: https://github.com/exercism/xfsharp/pull/318/files

Comment (jmbradnan) [inline docs/RESOURCES.md]: I like the F# example.     I will change and resubmit.    Should I also add info to the README.md at the same time?

Comment (ErikSchierboom) [inline docs/RESOURCES.md]: Yes please! That would be perfect

Comment (jmbradnan) [inline docs/RESOURCES.md]: I referenced https://gitter.im/exercism/xcsharp in my change, I am assuming this is correct.     There isn't any traffic over there yet which is why I mention it (could just be no one knows its there yet).     If I got this wrong though, let me know.    I believe this is right, so will go ahead and submit PR

Comment (jmbradnan) [inline docs/RESOURCES.md]: also, I forgot to put #208 in my commit message, but I *think* my pull request should be there.    First time I did this it kicked off automated tests, this time it did not so can't tell if I did this correctly, let me know if there is any issue.

**Comment by ErikSchierboom** [comment]
Great work @jmbradnan!

**Pull request merged**